api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -43,4 +43,140 @@ object IgnoredParameters {
     case "H2ODeepLearning" => common ++ Seq("pretrained_autoencoder")
     case _ => common
   }
+
+  def ignoredInMOJOs(algorithm: String): Seq[String] = {
+    val common = manuallyImplementedInMOJOModel ++ unimplementedForMOJOs
+    common ++ {
+      algorithm match {
+        case "H2OGBMMOJOModel" =>
+          Seq("max_categorical_levels", "huber_alpha")
+        case "H2ODRFMOJOModel" =>
+          Seq("tweedie_power", "max_categorical_levels", "quantile_alpha", "huber_alpha", "custom_distribution_func")
+        case "H2OXGBoostMOJOModel" =>
+          Seq(
+            "quantile_alpha",
+            "huber_alpha",
+            "max_categorical_levels",
+            "custom_metric_func",
+            "custom_distribution_func")
+        case "H2OGLMMOJOModel" =>
+          Seq(
+            "distribution",
+            "huber_alpha",
+            "max_categorical_levels",
+            "tweedie_power",
+            "quantile_alpha",
+            "gainslift_bins",
+            "categorical_encoding",
+            "custom_distribution_func")
+        case "H2OGAMMOJOModel" =>
+          Seq(
```

Comment:
@Pscheidl @wendycwong  Please could help me to identify parameters from the list that doesn't make any sense for GAM?